tensorflow/compiler/xla/service/all_reduce_reassociate.cc

```diff
@@ -122,13 +189,13 @@ StatusOr<bool> AllReduceReassociate::Run(
         continue;
       }
       // Find LHS all-reduce.
-      HloAllReduceInstruction* ar0 = LookThroughForAllReduce(
+      HloAllReduceInstruction *ar0 = LookThroughForAllReduce(
           inst->mutable_operand(0), *reduction_identity);
       if (ar0 == nullptr) {
         continue;
       }
       // Find RHS all-reduce.
-      HloAllReduceInstruction* ar1 = LookThroughForAllReduce(
+      HloAllReduceInstruction *ar1 = LookThroughForAllReduce(
```

Comment:
Can we go back to the earlier style where the * is with the type and not the variable name. I think this is the convention that XLA uses

Reply:
yea, not sure why clang formats the * with the variable name together. Let me change the rule and reformat.

Reply:
i see, the styling in this file is not consistent, some of the existing code has * in front of the variable and the formatter just follows that convention. I can make this consistent across the file.

Reply:
Hmm..Lets not do any formatting with the CL. I don't know why the style is not consistent.

Reply:
just saw the reply.. i already made the styling consistent, there were only a few places that needed to change.